internal/app/siftool/info.go

```diff
@@ -225,6 +231,14 @@ func writeInfo(w io.Writer, v sif.Descriptor) error {
 		}
 
 		fmt.Fprintf(tw, "\tFormat:\t%v\n", f)
+
+	case sif.DataOCIBlob:
+		t, err := v.OCIBlobMediaType()
+		if err != nil {
+			return err
+		}
+
+		fmt.Fprintf(tw, "\tType:\t%s\n", t)
```

Comment:
Nit: maybe make this more specific?

```suggestion
		fmt.Fprintf(tw, "\tMedia Type:\t%s\n", t)
```